src/lib/builtins/console.rs

```diff
@@ -94,6 +94,19 @@ fn log_string_from(x: Value) -> String {
             }
             s
         }
+        ValueData::Symbol(ref sym) => {
+            // Create empty formatted string to start writing to
+            let mut s = String::new();
+            let desc: Value = sym.borrow().get_internal_slot("Description");
+            match *desc {
```

Comment:
Why don't you have the match without the `write!`s, so it directly returns the result of the `format!`? This way you also don't need `s`.

Is there a specific reason?